.travis.yml

```diff
@@ -0,0 +1,59 @@
+language: generic
+
+sudo: required
+
+services:
+- docker
+
+env:
+  matrix:
+    - MOCK_CONFIG=fedora-rawhide-x86_64
+
+script:
+  - PKG=$(git log -1 --pretty=%s | sed 's/:.*//')
+  - if [[ ! -d ${PKG} ]]; then echo "Directory ${PKG} not found"; exit 1; fi
+  - if [[ ! -f ${PKG}/${PKG}.spec ]]; then echo "No spec file (${PKG}.spec found in directory ${PKG}"; exit 1; fi
+  - |
+    {
+      H="/travis/${PKG}"
+      echo "#!/bin/bash -xe"
+      echo "dnf -q -y install fedora-review"
+      echo "rpmbuild -D'_sourcedir ${H}' -D'_srcrpmdir ${H}' -bs ${H}/${PKG}.spec"
+      echo "useradd -d ${H} -g mock review"
+      echo "chown -R review:mock ${H}"
+      echo "/travis/spinner.sh \"su - -c 'fedora-review -v --mock-config ${MOCK_CONFIG} -n ${PKG}' review\" || { cat ${H}/.cache/fedora-review.log; find  ${H}/review-${PKG} -name '*.log' -print -exec cat {} \\;; exit 1; }"
```

Comment:
The whole `spinner.sh` stuff there is needed as `fedora-review` has no `--progress` option (see https://pagure.io/FedoraReview/issue/277) and Travis just dies after 10min without output. Also `fedora-review` won't print the log files if it fails.